libbeat/tests/integration/framework.go

```diff
@@ -481,7 +482,18 @@ func EnsureESIsRunning(t *testing.T) {
 	p, _ := esURL.User.Password()
 	req.SetBasicAuth(u, p)
 
-	resp, err := http.DefaultClient.Do(req)
+	skipSSL := false
+
+	if envESSSL := os.Getenv("ES_SKIP_SSL"); envESSSL == "1" {
+		skipSSL = true
+	}
+
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: skipSSL},
+	}
+	client := &http.Client{Transport: tr}
+
+	resp, err := client.Do(req)
```

Comment:
`ssl.verification_mode: none` is something you put in the output configuration, however here in `EnsureESIsRunning` no matter what you do in your output if the ElasticSearch you test against has an self-signed certificate, as the stack spawned by elastic-package, it will fail the test. Does that clear things up for you?

Reply:
reverted this commit